README.md

```diff
@@ -68,12 +68,14 @@ Request Body example:
                 'username': 'username',
                 'password': 'password',
                 'endpoint': 'url',
+                'auth': 'Kerberos',
                 'kernelname': 'pysparkkernel'
         }
         ```
 
-*Note that the kernelname parameter is optional and defaults to the one specified on the config file or pysparkkernel if not on the config file.*
+*Note: The auth can be either None, Basic_Access or Kerberos based on the authentication enabled in livy. kernelname parameter is optional and defaults to the one specified on the config file or pysparkkernel if not on the config file.*
```

Comment:
This is good for the README section on the server extension, but can you add a bullet point to the Features section above?